spec/features/user_views_list_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'User views list' do
+  scenario "and see's the list's books" do
+    list = create(:list_with_books, books_count: 2)
+
+    visit lists_path
```

Comment:
I'm thinking about this from the perspective that we aren't testing the routes in this spec but rather a snippet of a user behavior.  Is there a strong reason one way or the other to start at the `root_path` ?